config/config.go

```diff
@@ -209,22 +256,35 @@ type Server struct {
 	WriteTimeout   time.Duration `yaml:"write_timeout,omitempty"`
 }
 
+// Credential represents a credential value that may be a literal string or a file path.
+// To resolve the actual credential content, use conf.GetCredential(field).
+// File paths (starting with '/') are read with caching and automatic rotation support.
+type Credential string
+
+// String returns the credential value as a string.
+func (c Credential) String() string {
+	return string(c)
+}
```

Comment:
This seems pointless. If you need a string, can't you just do `string(cred)`?

Reply:
The purpose of this is to make Credential a fmt.Stringer; thus with this Credential can be used anywhere a fmt.Stringer is expected.